src/react/theme/globalStyles.js

```diff
@@ -33,5 +35,34 @@ export const setGlobalStyle = (theme = {}) => {
             background-color: ${headerTheme.getBgColor()};
             color:  ${headerTheme.getTextColor()};
         }
+
+        .oskari-root-el > nav {
+            .scroll-indicator {
+                display: none;
+                background-color: ${navColor};
+                border-top: ${headerTheme.getAccentColor()} solid 2px;
+                height: 45px;
+                width: ${navigationDimensions.width}px;
+                pointer-events: none;
+                position: absolute;
+                left: ${navigationDimensions.left}px;
```

Comment:
Isn't left always 0 since the scroll indicator is inside the nav? Now if the navigation is on the right side the indicator will be off screen?